src/NftDetails/NftDetails.tsx

```diff
@@ -26,7 +27,7 @@ export const NftDetails = () => {
   return (
     <FadeIn style={styles.container}>
       <ScrollView contentContainerStyle={styles.contentContainer}>
-        <NftImage nft={nft} />
+        {MODERATING_NFTS_ENABLED ? <ModeratedNftImage nft={nft} /> : <UnModeratedNftImage nft={nft} />}
```

Comment:
I think eslint is going to complain if `MODERATING_NFTS_ENABLED === true` is not used